servicetalk-http-api/src/main/java/io/servicetalk/http/api/StreamingHttpResponse.java

```diff
@@ -122,11 +135,26 @@
     /**
      * Returns a {@link StreamingHttpResponse} with its underlying payload transformed. Note that the raw objects of the
      * underlying {@link Publisher} may be exposed. The object types are not guaranteed to be homogeneous.
+     * @deprecated Use {@link #transformPayloadBody(UnaryOperator)}.
      * @param transformer Responsible for transforming the payload body.
      * @return {@code this}
      */
+    @Deprecated
     StreamingHttpResponse transformRawPayloadBody(UnaryOperator<Publisher<?>> transformer);
```

Comment:
This method is still used in `ContentLengthTest`, please update the test. Also, referenced in javadoc of both tracing filters.

Reply:
Lets discuss on https://github.com/apple/servicetalk/pull/1332#discussion_r565782546 due to overlapping discussion.

Reply:
Makes sense for tests, let's just update javadoc in `TracingHttpRequesterFilter` and `TracingHttpServiceFilter`

Reply:
Just a reminder about this thread

Reply:
+1 good catch!